client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -346,4 +351,13 @@ class TransferDomainPrecheck extends React.PureComponent {
 	}
 }
 
-export default localize( TransferDomainPrecheck );
+const recordNextStep = ( domain_name, show_step ) =>
```

Comment:
You don't really need these if it's only a single event.

You can just pass `recordTracksEvent` to the `mapDispatchToProps` and call them directly in the code. :) Your call.